infrastructure/requirements.txt

```diff
@@ -1,5 +1,8 @@
-aiogram~=3.0
 environs~=9.0
 uvicorn
 fastapi
 betterlogging
+sqlalchemy~=2.0
```

Comment:
i don't think we need to have a single requirements for all infrastructure, i'd better leave it inside api folder